net/zerotier/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		zerotier
-PLUGIN_VERSION=		1.0
+PLUGIN_VERSION=		1.1.0
```

Comment:
Any reason for the extra digit ".0" ? We also have PORTREVISION= X which sets the version of the package to 1.1.0_X, it might be confusing to talk about the correct version with more digits. Just for info. :)

Reply:
Hi,

I was trying to adhere to the semver.org system of major.minor.patch. Is that okay?